manifests/endpoint-observability/observability.open-cluster-management.io_observabilityaddon_crd.yaml

```diff
@@ -44,21 +44,14 @@ spec:
                   metrics to hub server.
                 type: boolean
               interval:
-                default: 30
+                default: 60
                 description: Interval for the observability addon push metrics to
                   hub server.
                 format: int32
                 maximum: 3600
                 minimum: 15
                 type: integer
               resources:
-                default:
-                  limits:
-                    cpu: 100m
-                    memory: 600Mi
-                  requests:
-                    cpu: 100m
-                    memory: 100Mi
                 description: Resource requirement for metrics-collector
```

Comment:
Are these changes intent?

Reply:
Yes. it should keep consistent with https://github.com/open-cluster-management/multicluster-observability-operator/blob/release-2.3/config/crd/bases/observability.open-cluster-management.io_observabilityaddons.yaml